reliability.md

```diff
@@ -1,33 +1,45 @@
+
+On the other hand, if you are a media outlet that publishes a hundred articles a day and there is a lot of interaction, you will probably want to make a copy every few hours.
+
+In any case, it is advisable to have at least one daily copy for a week, and then store a weekly copy for at least a few months. This way you'll be sure to have copies that go back 3-6 months. Again, this will depend on the size and configuration of your site.
+
+## Monitoring
 
-### Performance Monitoring
+Having monitoring tools will allow you to check if the site is working properly, at many different levels.
 
-While a site's services may be responding, to a user, a site being "up" means more than this to them. Performance monitoring is similar to uptime monitoring, but also takes note of certain metrics that could indicate trouble. Metrics like "page load time" and "slowest average transactions" should be monitored and reported regularly to help keep you ahead of performance issues. Monitoring slow logs for problematic queries or requests can also help keep user sites stable. MySQL, PHP-FPM, and others provide options to capture these for monitoring.
+### Uptime
 
-### Performance Profiling
+One of the basic monitoring systems is to know that the site is active, in addition to small elements of loading speed or response times. Usually you use external ping services that call your website frequently and check that it is working and online, and whether access is fast or slow. In case there is a problem, it usually sends some kind of warning as an alert.
 
-It is best practice to use performance profiling tools, such as New Relic, AppDynamics or Tideways, to diagnose the performance bottlenecks of your website and infrastructure. These tools will give you insight such as slow performing functions, external HTTP requests, slow database queries and more that are causing poor performance.
+In addition, these services usually have panels in which they can analyze these times and possible service failures in a historical way.
 
-## Version Control
+### Performance
 
-Version control is a way of tracking the changes made to files over time by different people, such as the code for a website or another application. It allows people to track the revision history of code and to revert or apply changes easily via the command line. It is also a good way to debug your website if something goes wrong, as you can quickly restore to a previous state of the site's code without restoring from a full backup.
+In general, the measurement of these data is usually done internally as they mainly affect the services available to the machine. For a WordPress to work we need at least the server with its operating system, the web server, the PHP and the database. In addition, we can have the caching systems. Well, the performance monitoring system analyzes all this.
```

Comment:
This is less clear to me than the previous descriptions in lines 19 and 23.

Those are terse, so they could use a bit of reworking, but maybe we could start there?